railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -52,6 +52,7 @@ group :development do
   # gem "spring"
 <%- if RUBY_VERSION >= "3.1" && RUBY_VERSION < "3.2" -%>
 
+  # Add short explanation where the exception is raised [https://github.com/ruby/error_highlight]
```

Comment:
To me the wording used in the PR is a bit more descriptive:
```suggestion
  # Show the fine-grained location where an exception is raised [https://github.com/ruby/error_highlight]
```

Or maybe use 'error' instead of 'exception' to use the same naming convention as the gem:
```suggestion
  # Show the fine-grained location where an error occured [https://github.com/ruby/error_highlight]
```

Or even use 'highlight':
```suggestion
  # Highlight the fine-grained location where an error occured [https://github.com/ruby/error_highlight]
```

Reply:
Thanks @p8 , Third one is looks more appropriate.